muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/reducers/workspaces/index.ts

```diff
@@ -373,13 +375,13 @@ export interface MaterialAssignmentType {
   assignmentType: "EXERCISE" | "EVALUATED",
   correctAnswers: string,
   path: string,
-  title: string
+  title: string,
 }
 
 export interface MaterialContentNodeProducerType {
-  id: number;
-  name: string;
-  materialId: number;
+  id: number,
+  name: string,
+  materialId: number,
```

Comment:
Why comma after last element?

Reply:
Same reason, for diffing purposes and a stylistic preference.

Now ";" is preferred but we used comma when making muikku because that was what we had then, and it's kept that way.